src/localization/en.json

```diff
@@ -167,7 +167,8 @@
 
   "exhibition": {
     "reception": "Reception",
-    "visitorVariablesManagement": "Visitor variables management",
+    "visitorVariables": "Visitor variables",
```

Comment:
Visitor variables and visitor varialbe management sound quite similar, should this be called, e.g. Variable management?